new-project/tests/login.spec.ts

```diff
@@ -0,0 +1,9 @@
+import { test, expect } from '@playwright/test';
+
+test('test', async ({ page }) => {
```

Comment:
It's better to get used to call your test scenario by the functionality which it tests.